routers/static_filter.go

```diff
@@ -38,9 +42,70 @@ func StaticFilter(ctx *context.Context) {
 		path += urlPath
 	}
 
+	CheckStaticUrl()
+
 	if util.FileExist(path) {
 		http.ServeFile(ctx.ResponseWriter, ctx.Request, path)
```

Comment:
Implement `ServeFileWithReplace(ctx.ResponseWriter, ctx.Request, path, "old-string", "new-string")`

- Don't list files in folder
- Don't write files